Resources/Textures/Objects/Misc/ice_crust.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from space-station-14+Resources+Textures+Tiles+Planet+Snow",
```

Comment:
use the copyright listed in the attributions.yml file for the ice floor.

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/59531932/685d2ee7-5296-4827-aaac-e107f830ea2e)
I would do that, but I've not found atributions.yml file in that folder and meta.json has no info :)

Commit where this file was added did not include any copyright claims as well:
https://github.com/space-wizards/space-station-14/pull/13487/files#diff-22a6f3bc9bafb71b461840aa1ebfb0f7f625ef81339732f58b7880111aaeb4d9